include/cppoptlib/solver/cmaesbsolver.h

```diff
@@ -0,0 +1,231 @@
+// CppNumericalSolver
+#ifndef CMAESB_H
+#define CMAESB_H
+
+#include <random>
+#include <Eigen/Dense>
+#include "isolver.h"
+
+namespace cppoptlib {
+
+/**
+ * @brief Covariance Matrix Adaptation
+ */
+template<typename TProblem>
+class CMAesBSolver : public ISolver<TProblem, 1> {
+public:
+    using Super = ISolver<TProblem, 1>;
```

Comment:
It's defined here as a typedef (with C++11 syntax). I think it makes the intent a bit clearer.